shared-helpers/src/types/backend-swagger.ts

```diff
@@ -1580,7 +1580,7 @@ export class UserService {
   /**
    * List users in CSV
    */
-  listAsCsv(options: IRequestOptions = {}): Promise<SuccessDTO> {
+  listAsCsv(options: IRequestOptions = {}): Promise<any> {
```

Comment:
Is there a reason this wasn't generated to be StreamableFile?

Reply:
Good question. This typing is picked up from the `@ApiOkResponse()` field which we aren't adding to the `listAsCsv` endpoints. I decided to keep this in sync with the other csv endpoints. But that is something we can look to update in the future